src/main/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsController.java

```diff
@@ -55,7 +58,37 @@ public ResponseEntity<ServiceHearingValues> serviceHearingValues(
 
             ServiceHearingValues model = serviceHearingsService.getServiceHearingValues(request);
 
-            log.info("serviceHearingValues response {}", model);
+            var modelForLogging =  List.of(
```

Comment:
do we need to log everything? 
and it seems like the only thing that needs masking is the party details, so why not get a copy of the model object, and just change the party details